test/lit.cfg

```diff
@@ -1340,6 +1340,76 @@ elif run_os == 'linux-androideabi' or run_os == 'linux-android':
         '-L%s' % make_path(test_resource_dir, config.target_sdk_name)])
     # The Swift interpreter is not available when targeting Android.
     config.available_features.discard('swift_interpreter')
+elif run_os == 'wasi':
+    tools_directory = pipes.quote(make_path(config.wasi_sdk_path, "bin"))
+
+    lit_config.note("Testing WebAssembly/WASI " + config.variant_triple)
+
+    config.target_object_format = "wasm"
+    config.target_shared_library_prefix = 'lib'
+    config.target_shared_library_suffix = ".a"
+    config.target_sdk_name = "wasi"
+    config.target_runtime = "native"
+
+    config.target_swift_autolink_extract = inferSwiftBinary("swift-autolink-extract")
+
+    config.target_build_swift = ' '.join([
+        '%s', '-target %s',
+	'-Xcc --sysroot=%s', '-Xclang-linker --sysroot=%s',
```

Comment:
Similar for these two.  These three are effectively no-ops as they will be ignored as they are treated as a single quoted group.  If it is working, it is happening to work due to invalid quoting.